File: LB-Assignments/Assignment_31/program1.c
/*
Write a program which accepts string from user 
and count number of capital characters
*/
#include<stdio.h>

int CountCapital(char *str)
{
    int iCount = 0;
    while(*str != '\0')
    {
        if(*str >= 'A' && *str <= 'Z')
        {
            iCount++;
        }
        str++;
    }
    return iCount;

}
int main()
{
    char arr[20] ={'\0'};
    int iRet = 0;

    printf("Enter String : ");
    scanf("%[^'\n']s",arr);

    iRet = CountCapital(arr);

    printf("Capital Characters are : %d",iRet);

    return 0;
}

File: LB-Assignments/Assignment_38/program1.java
/*
Accept number from user and count number of
on bits without using % and /
*/
import java.util.*;

class Bitwise
{
    public int iNo;

    public Bitwise(int a)
    {
        this.iNo = a;
    }

    public int CountOn()
    {
        int iCount = 0;
        int iTemp = iNo;

        while(iTemp != 0)
        {
            if((iTemp & 1)==1)
            {
                iCount++;
            }
            iTemp = iTemp >> 1;
        }
        return iCount;

    }
}
class program1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0, iRet = 0;

        System.out.println("Enter Number : ");
        iValue = sobj.nextInt();

        Bitwise bobj = new Bitwise(iValue);

        iRet = bobj.CountOn();

        System.out.println("number of On bits are : "+iRet);

    }
}